policyengine_uk/parameters/gov/hmrc/tax-free-childcare/tax-free-childcare-benefits.py

```diff
@@ -0,0 +1,53 @@
+from policyengine_uk.model_api import *
+
+class tax_free_childcare_amount(Variable):
+    value_type = float
+    entity = BenUnit
+    label = "Tax-free childcare support amount"
+    definition_period = YEAR
+    unit = GBP
+
+    def formula(benunit, period, parameters):
+        p = parameters(period).gov.tax_free_childcare
+
+        # Check eligibility conditions from your YAML files
+        meets_age_condition = benunit("child_age_eligible", period)
```

Comment:
is this variable defined in the UK repository - if not we need to add a separate class checking the age against a threshold which is added as a separate yaml file

Reply:
similar question for other eligibility conditions